chimney/src/main/scala/io/scalaland/chimney/internal/compiletime/Contexts.scala

```diff
@@ -75,12 +89,19 @@ private[compiletime] trait Contexts { this: Definitions & Configurations =>
       def create[From: Type, To: Type](
           src: Expr[From],
           failFast: Expr[Boolean],
-          config: TransformerConfig
+          config: TransformerConfig,
+          runtimeDataStore: Option[Expr[TransformerDefinitionCommons.RuntimeDataStore]]
       ): ForPartial[From, To] = ForPartial(
         From = Type[From],
         To = Type[To],
         src = src,
         failFast = failFast,
+        runtimeDataStore = runtimeDataStore.getOrElse(
+          Expr.asInstanceOf[Nothing, TransformerDefinitionCommons.RuntimeDataStore](Expr.Nothing)(
+            Type.Nothing,
+            ChimneyType.RuntimeDataStore
+          )
+        ),
```

Comment:
Here, I am also on the edge. I would like to prevent setting up some stubs when we aren't using `RuntimeDataStore` but at the same time I want to be able to assume that when we're using it, we are not shoving `Option`s  back and forth - I guess when we'll start using it directly we can try out a few things and see what works best (`null` as default, assert != null on first use, `???` expr as default which would not be used etc).

Reply:
We can always expr-ize `TransformerDefinitionCommons.emptyRuntimeDataStore` as empty collection expr and keep the context simple, without options.